packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -29,18 +54,24 @@
     <h1>THIS IS A WORK IN PROGRESS SAMPLE</h1>
     <section>
       <h2>Meetings</h2>
-      <p>[[APPLICATION DESCRIPTION]]</p>
+      <p>This is the meetings plugin sample for the Webex JS SDK.</p>
+      <p>Each of the following sections within this sample contain forms that perform specific actions within the meetings plugin of the Webex JS SDK.</p>
+      <p>To find the exact logic performed by each of the form input elements, please inspect the browser's developer console, as it will provide a reference to a function in this application that performs the selected action within the ./app.js file.</p>
     </section>
 
     <section>
```

Comment:
This is what I'm suggesting each section should look like
It's pretty much what 90% of CSS framework docs are setup as

```html
<section>
  <h2>Authentication</h2>
  <p>This section contains all of the logic needed to fully authenticate and prepare the Webex JS SDK for meetings usage. Before using the meetings functionality of the Webex JS SDK, the Webex JS SDK must be initialized using a valid token, and once ready, registered as a device.</p>
  <p>There are two steps that need to be performed before the meetings functionality of the JS SDK is available for use are:</p>
  <ul>
  	<li>Webex.init() =&gt; Initialize the Webex JS SDK</li>
    <li>webex.meetings.register() =&gt; Register the local device using</li>
  </ul>
  <p>Once the Webex JS SDK instance is ready and registered, The rest of the meetings functionality of the Webex JS SDK becomes available. These values can be confirmed by checking:</p>
  <ul>
    <li>webex.ready =&gt; Determine if the webex instance is ready</li>
    <li>webex.meetings.registered =&gt; Determine if the webex instances local device is registered</li>
  <form id="credentials">
    <fieldset>
      <legend>Credentials</legend>
      <input id="access-token" name="accessToken" placeholder="Access Token" value="" type="text">
      <button id="access-token-save" type="button" onclick="saveCredentials()">Save</button>
      <label id="access-token-status"></label>
    </fieldset>
  </form>
  <br>
  <form lpformnum="1">
    <fieldset>
      <legend>Registration</legend>
      <button id="registration-register" type="button" onclick="register()" disabled="">Register</button>
      <button id="registration-unregister" type="button" onclick="unregister()" disabled="">Unregister</button>
      <label id="registration-status">Not Registered</label>
    </fieldset>
  </form>
  <div class="collapsible" data-collapsed="false">Code Sample</div>
    <div class="collapsible-content" data-collapsed="false">
    <code wrap="false">
      let webex = (window.webex = Webex.init({
        credentials: {
          access_token: <YOUR ACCESS TOKEN HERE>,
        },
      }));

      webex.once('ready', () => {
        webex.meetings.register()
          .then(() => {
            console.log('Authentication#register() :: successfully registered');
          })
          .catch((error) => {
            console.warn('Authentication#register() :: error registering', error);
          });
      });
	</code>        
  </div>
</section>
```

Reply:
Leaving this as is for now, we will be adding features as described in this comment in a later PR.